### ISSUE TITLE
Rule 932370 has false positive for "At" after newline

### ISSUE THREAD

**Issue by ssigwart** [coreruleset/coreruleset#3953]
### Description

"At" after a newline triggers rule 932370. For example, this:
```
Go to store.
At the store, shop.
```

https://github.com/coreruleset/coreruleset/blob/44b82683028188f770541567db760e24810e84ca/regex-assembly/932370.ra#L37

### How to reproduce the misbehavior (-> curl call)

```sh
curl -H "x-format-output: txt-matched-rules" "https://sandbox.coreruleset.org/?msg=Go%20to%20store.%0AAt%20the%20store%2C%20shop."
```

```
932370 PL1 Remote Command Execution: Windows Command Injection
949110 PL1 Inbound Anomaly Score Exceeded (Total Score: 5)
980170 PL1 Anomaly Scores: (Inbound Scores: blocking=5, detection=5, per_pl=5-0-0-0, threshold=5) - (Outbound Scores: blocking=0, detection=0, per_pl=0-0-0-0, threshold=4) - (SQLI=0, XSS=0, RFI=0, LFI=0, RCE=5, PHPI=0, HTTP=0, SESS=0, COMBINED_SCORE=5)
```

### Your Environment

* CRS version (e.g., v3.3.4): 4.8.0
* Paranoia level setting (e.g. PL1) : PL1
* ModSecurity version (e.g., 2.9.6): 3.0.13
* Web Server and version or cloud provider / CDN (e.g., Apache httpd 2.4.54): Nginx 1.26.1
* Operating System and version: Linux

### Confirmation

- [X] I have removed any personal data (email addresses, IP addresses,
    passwords, domain names) from any logs posted.

**Comment by airween** [comment]
Hi @ssigwart,

thanks for letting know this issue. As I remember there was a discussion about the word "at", but it seems that finally we added that. It's sad that rule is on PL1.

Unfortunately what you can do is that create an exclusion against that FP. Feel free to ask help if you need.

**Comment by ssigwart** [comment]
Thanks, @airween. I can create an exclusion. I just try to report things here in case others have similar issues.

**Comment by prologic** [comment]
This rule is also a false positive for simple posts to a web app ([yarn](https://git.mils.io/yarnsocial/yarn)):

```
"msg": "Remote Command Execution: Windows Command Injection",
10 | "data": "Matched Data: ` at all found within ARGS:text: @kat@yarn.girlonthemoon.xyz Btw, as a new pod operator, please do let me know how I can improve `yarnd` at all 🙏\n",
```

This is really disappointing actually. I'm tempted to drop this rule entirely from my ruleset.

**Comment by prologic** [comment]
I've now added this to the end of my config (_after loading the OWASP ruleset_):

```
SecRuleRemoveById 932370
```

**Comment by theseion** [comment]
@EsadCetiner Can we close this one two after having merged #3735?

**Comment by EsadCetiner** [comment]
@airween this one's for Windows, not Unix. I can exclude `as` and `at` if you want just as was done for the unix rules.

**Comment by EsadCetiner** [comment]
sorry, I meant to tag @theseion apologies

**Comment by theseion** [comment]
Right. Would be great if you could make that change, if necessary. `at` is suffixed with `@`, probably doesn't make a difference in this case.
